portfolio/src/main/webapp/index.html

```diff
@@ -6,48 +6,107 @@
+                </div>
+                <div class="project-text">
+                  <p>During my sophomore year of high school I was a part of planning our school's first hackathon, InspiredxSTEM.
+                    I assisted with advertising the event and build a simple webpage that gives visitors an introduction of
+                    what the event is as well as a link to RSVP for the event. During the event itself I served as a mentor
+                    to students ranging from middle school to early high school as they were working on their projects.</p>
+                  <a href="https://inspiredxstem.github.io/inspired/" target="blank">Link to website (still working as of 6/6/2020)</a>
+                </div>
+              </div>
+              <div class="project-box">
+                <div class="project-image">
+                  <h2>Animoto</h2>
+                  <img src="/images/animoto.jpg">
+                </div>
+                <div class="project-text">
+                  <p>During the summer of 2017 I was given the opportunity to be a paid intern at Animoto, a cloud-based video creation service.
+                    throughout my time at this company I was able to not only gain a significant number of meaningful connections in the
+                    web development field, I also gained experience in working on large scale projects through git as well as writing tests and
+                    completing a redesign of the video editor (which many of its features are still live on the site).</p>
+                </div>
+              </div>   
+              <div class="buttons">
+                <button onclick="swapHidden('projects','intro-main')">Return To Main Page</button>
+                <button onclick="swapHidden('projects','gallery')">Gallery</button>
+                <button onclick="swapHidden('projects','interests')">Interests</button>
+              </div>
+            </div>
+            <!--Below is the code for the gallery page and its grid.-->
+            <div id="gallery">
+              <h1>Gallery</h1>
+              <p>Click on the photos below for some context on when these were taken!</p>
+              <div id="gallery-grid">
+                <img src="/images/posse.jpg">
+                <img src="/images/sammy.png">
+                <img src="/images/inspiredxSTEM.jpg">
+                <img src="/images/animoto.jpg">
+              </div>
+              <div class="buttons">
+                <button onclick="swapHidden('gallery','intro-main')">Return To Main Page</button>
+                <button onclick="swapHidden('gallery','projects')">Projects</button>
+                <button onclick="swapHidden('gallery','interests')">Interests</button>
+              </div>
+            </div>
+            <!--Below is the code for the "Interests" page. Due to time constraints, this page isn't implemented but the code is left here so that I can complete
+                later.-->
```

Comment:
nit: For things like this where you're commenting on work that needs to be done, I'd recommend using TODO comments (e.g. `<!--TODO: Implement interests page.-->`. A lot of IDEs have TODO comment integrations so it makes it easier to manage.